app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError
+from app.errors import VaccineError
+
+
+def go_to_cafe(cafe: Cafe, friends: list) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except VaccineError:
+            return "All friends should be vaccinated"
+    if masks_to_buy != 0:
```

Comment:
Don't compare a variable to 0 if there is no need

Reply:
![image](https://user-images.githubusercontent.com/103201010/219680874-b4e32f29-1db1-4165-84ad-3f20ec7e5fa7.png)
@viktoria-rybenchuk is right, fix this please